### PR TITLE
:bug: fix urlPrefix on SeriesTable

### LINKED ISSUE #4562: Stats page wrong link
### What happened?

Links on galleries in https://kodadot.xyz/series-insight point to rmrk instead of bsx, leading to 404.

### Please reproduce in steps

Go to https://kodadot.xyz/series-insight click on any collection name.

### Expected Behavior

Points to bsx collections

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4562
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/series/SeriesTable.vue [hunk 1]
```diff
@@ -396,6 +398,10 @@ export default class SeriesTable extends mixins(PrefixMixin, AuthMixin) {
     await this.fetchCollectionsSeries(Number(this.nbRows))
   }
 
+  get urlPrefix() {
+    return this.$store.getters.currentUrlPrefix
+  }
```
Comment (Jarsen136) [inline components/series/SeriesTable.vue:403]: We would not need this function cause we could get the `urlPrefix` from the existing `PrefixMixin`

Comment (petersopko) [reply components/series/SeriesTable.vue]: didn't notice that's part of it, fixed now.

**Review by Jarsen136 — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 22.64 USD/KSM ~ 2.208 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x8def7be34c5581e1b0a26f37c327f497e10c938b89611466f4f536e913bc8c2c](https://kusama.subscan.io/extrinsic/0x8def7be34c5581e1b0a26f37c327f497e10c938b89611466f4f536e913bc8c2c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**